tests/acceptance/features/apiOcm/share.feature

```diff
@@ -753,3 +753,45 @@ Feature: an user shares resources using ScienceMesh application
     Then the HTTP status code should be "200"
     And the downloaded image should be "32" pixels wide and "32" pixels high
     And the downloaded preview content should match with "thumbnail.png" fixtures preview content
+
+  @issue-10213
+  Scenario: local user deletes access of shared resource to federated user
+    Given using spaces DAV path
+    And using server "LOCAL"
+    And user "Alice" has created a folder "FOLDER" in space "Personal"
+    And "Alice" has created the federation share invitation
+    And using server "REMOTE"
+    And "Brian" has accepted invitation
+    And using server "LOCAL"
+    And user "Alice" has sent the following resource share invitation to federated user:
+      | resource        | FOLDER   |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    When user "Alice" removes the access of user "Brian" from resource "FOLDER" of space "Personal" using the Graph API
+    Then the HTTP status code should be "204"
+    When using server "REMOTE"
+    And for user "Brian" the space "Shares" should not contain these entries:
```

Comment:
I have a doubt on this step. Could you check if this step works without deleting the share

Reply:
yup the step is passing even without deleting the share.
Should we check the response or make another step to check for federated shares?